### PR TITLE
vbaProject should use TargetFileExtension .bin

### PR DESCRIPTION
**Pull request by mikeebowen**

### CODE REVIEW

**Review by twsouthwick — APPROVED** [review]

**Comment by tomjebo** [comment]
draft, excellent.  explanation: we are verifying and doing some testing to make sure this is correct before. I believe it is. @hunyu can update here.

**Comment by hunyu** [comment]
> draft, excellent. explanation: we are verifying and doing some testing to make sure this is correct before merging. I believe it is. @hunyu can update here.

This is in regarding to the issue 1292 that was raised - https://github.com/OfficeDev/Open-XML-SDK/issues/1292

**Comment by tomjebo** [comment]
@hunyu also, we need to look into the vbaprojectdata part to make sure that uses the correct extension.

**Comment by twsouthwick** [comment]
FYI this should also be ported to the release/v2.20.0 branch once it's finalized here

**Comment by twsouthwick** [comment]
Can you update the CHANGELOG?

**Comment by twsouthwick** [comment]
@mikeebowen is this ready to be out of draft? Is there a blocker here?

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/Open-XML-SDK?pullRequest=1301) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/Open-XML-SDK?pullRequest=1301) before we can accept your contribution.<br/><br/>:x: mikeebowen [sign now](https://cla.dotnetfoundation.org/dotnet/Open-XML-SDK?pullRequest=1301)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/Open-XML-SDK?pullRequest=1301) it.</sub>

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,11 +31,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
+### Fixed
+- Fixed incorrect file extensions for vbaProject files (#1292)
+
 ## [2.20.0]
 
 ### Changed
 - Marked all obsoleted APIs as compile as error. These will be removed in the next release
 
+### Fixed
```
Comment (twsouthwick) [inline CHANGELOG.md:42]: nit: in the future, you don't need to worry about update 2.20 on main; we'll copy the lisitng from that branch over to here

**Review by twsouthwick — APPROVED** [review]

**Comment by twsouthwick** [comment]
@mikeebowen once you've signed the CLA (must be the first time you're contributing in the dotnet org) we can merge it in

**Comment by tomjebo** [comment]
@mikeebowen I had asked about the vbadatapart but doesn't look like anyone looked into it. Can you take a look and see if we need to add this as well?

**Comment by mikeebowen** [comment]
@tomjebo vbaDataPart is an xml part from Word documents with VBA macros, so no change needed. [VBA Supplemental Data Part of a Word-Processing Document](https://learn.microsoft.com/en-us/openspecs/office_standards/ms-offmacro/c82ae51e-ea14-4051-b71f-3b69ddb011c2)

**Comment by tomjebo** [comment]
thanks for checking.

**Pull request merged**